README.md

```diff
@@ -88,6 +88,66 @@ cat spec/fixtures/bb112zx3193.json | janeway "$.description.contributor[?@.role[
 ]
 ```
 
+### Formatting for display
+
+Methods ending in `_display_data` usually return arrays of a class called `DisplayData`, which is designed for rendering into HTML by a consuming application. Each `DisplayData` object has a `label`, which serves as a heading under which its data is grouped. The `values` method returns an array of strings, which are the individual values to be displayed under that heading.
+
+For example, when displaying contributors, the label is usually determined by the role of the contributor, and the values are the display names of the contributors with that role:
+
+```ruby
+> rec.contributor_display_data.first.label
+=> "Former owner"
+> rec.contributor_display_data.first.values
+=> ["Hearst Magazines, Inc."]
+```
+
+The `#to_hash` helper method, which collapses all provided `DisplayData` into a single hash, can be used as a quick way to check the overall structure:
+
+```ruby
+> CocinaDisplay::DisplayData.to_hash(record.subject_display_data)
+=> {"Marque"=>["Bugatti"], "Model"=>["Bugatti T51A"], "Subject"=>["Bugatti automobile"]}
+```
+
+Note that usage of the `displayLabel` attribute in Cocina overrides the `label` that would ordinarily be used to group an item. It's possible for use of `displayLabel` to result in items being grouped together after relabeling, or to be split out into their own group.
```

Comment:
I am not sure I totally understand what this sentence is trying to say despite knowing the context of displayLabel

Reply:
I took another shot at this but it might just be too confusing inherently, lol